.github/workflows/turing.yaml

```diff
@@ -58,6 +58,13 @@ jobs:
         with:
           fetch-depth: 0
 
+      # This is needed due to the issue raised here: https://github.com/golang/go/issues/61455
+      - name: Install Go ${{ env.GO_VERSION }}
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          cache: false
```

Comment:
Not having this step causes the following errors to be thrown when `go mod vendor` is run in the CICD runners:
```
go: downloading go1.22 (linux/amd64)
go: download go1.22 for linux/amd64: toolchain not available
```
This issue is mentioned in https://github.com/golang/go/issues/61455, and one of the workarounds is to simply explicitly install go 1.22 https://github.com/shogo82148/actions-goveralls/issues/485.